smartsim/_core/launcher/step/alpsStep.py

```diff
@@ -120,24 +142,24 @@ def _build_exe(self) -> t.List[str]:
         :return: executable list
         :rtype: list[str]
         """
-        if self.run_settings.mpmd:
+        if self._get_mpmd():
             return self._make_mpmd()
 
         exe = self.run_settings.exe
-        args = self.run_settings.exe_args
+        args = self.run_settings._exe_args  # pylint: disable=protected-access
         return exe + args
 
     def _make_mpmd(self) -> t.List[str]:
         """Build Aprun (MPMD) executable"""
 
         exe = self.run_settings.exe
-        exe_args = self.run_settings.exe_args
+        exe_args = self.run_settings._exe_args  # pylint: disable=protected-access
```

Comment:
note to self: go back & look @ when exe_args was added (was it during mitigation of mypy?). if not in open use, consider changing the signature to avoid the t.Union[str, t.List[str]] and define `def set_exe_args`

Reply:
reviewed..